### DISCUSSION TITLE
Unable to get MS Code Coverage to use my runsettings

### DISCUSSION THREAD

**Discussion by mjarosch** [FortuneN/FineCodeCoverage#344] [Q&A]
I am having troubles getting the ms code coverage setup and working on a new project. I have a single test project that I have added the  Microsoft.CodeCoverage nuget reference. I have created a CodeCoverage.runsettings file in the root, next to my .sln file. I added the TestAdaptersPaths to my runsettings, trying both the path to the nuget and the FCC provided one. It continues to generate a runsettings file, ignoring my file.

Is there a sample repo somewhere where I can see it setup and working? My project is in a "src" folder, does that additional hierarchy affect it?

**Comment by tonyhallett** [comment]
Have you told visual studio to use your runsettings file ?

**Reply by mjarosch** [reply]
I have it set to autodetect the runsettings file. Do I need to specifically select it?

**Reply by tonyhallett** [reply]
No not if it conforms to https://learn.microsoft.com/en-us/visualstudio/test/configure-unit-tests-by-using-a-dot-runsettings-file?view=vs-2022 that states the file should be named .runsettings

**Reply by mjarosch** [reply]
Is it .runsettings or *.runsettings? The linked docs say both. Mine is named CodeCoverage.runsettings.

![image](https://github.com/FortuneN/FineCodeCoverage/assets/79384/e071e7eb-9029-490d-bfb4-3290bb8e7180)

**Reply by mjarosch** [reply]
Ok, missed that note where the autodetect needs to be named .runsettings. After fixing that and couple of other things, I think I have it working.
